lib/org/chromium/webidl/ast/Attribute.js

```diff
@@ -32,6 +32,7 @@ foam.CLASS({
     },
     function outputWebIDL(o) {
       if (this.isInherited) o.outputStrs('inherit ');
+      if (this.isStatic) o.outputStrs('static ');
```

Comment:
`isInherited` and `isStatic` are mutually exclusive (i.e. they cannot both be true at once), thus the order of output should not matter.